src/main/java/picard/util/LiftOverIntervalList.java

```diff
@@ -41,6 +41,32 @@
 import java.util.List;
 
 /**
+ *
+ * This tool adjusts the coordinates in an interval list on one reference to its homologous interval list on another
+ * reference, based on a chain file that describes the correspondence between the two references. It is based on the
+ * UCSC LiftOver tool (see: http://genome.ucsc.edu/cgi-bin/hgLiftOver) and uses a UCSC chain file to guide its operation.
+ * It accepts both Picard interval_list files or VCF files as interval inputs.
+ * <br />
+ * <h3>Usage example:</h3>
+ * <pre>
+ * java -jar picard.jar LiftOverIntervalList \\
+ *       I=input.interval_list \\
+ *       O=output.interval_list \\
+ *       SD=reference_sequence.dict \\
+ *       CHAIN=build.chain
+ * </pre>
+ *
+ * <h3>Return codes</h3>
+ * If all the intervals lifted over successfully, program will return 0. It will return 1 otherwise.
+ *
+ * <h3>Caveats</h3>
+ * Intervals are "lifted" in their entirety, but they might intersect (a "hit") with multiple chain-blocks which would attempt to
+ * place it in different places. In this case, The interval is lifted over using the first hit that passes the threshold
```

Comment:
What is the antecedent of "it" in the first sentence? Maybe ...
``` java
/**
...
* An interval is "lifted" in its entirety,
* but it might intersect (a "hit") with multiple chain-blocks.
* Instead of placing the interval in multiple hits,
* it is lifted over using the first hit
* that passes the threshold of {@link #MIN_LIFTOVER_PCT}.
...
*/
```
Warning: I barely misunderstand this stuff.